ansible_collections/arista/avd/molecule/eos_cli_config_gen/inventory/host_vars/router-pim-sparse-mode.yml

```diff
@@ -32,3 +36,15 @@ router_pim_sparse_mode:
       ipv4:
         rp_addresses:
           - address: 10.238.3.161
+    - name: Test_RP_ACL
+      ipv4:
+        rp_addresses:
+          - address: 10.238.4.161
+            access_lists:
```

Comment:
let's add one test where we render neither group nor access-list but some options like priority, ..

Reply:
Added in https://github.com/aristanetworks/ansible-avd/pull/2355/commits/f7d64aebe328e700b083143a62c1234cf3b8c5f1